pkg/sdk/internal/random/random_helpers.go

```diff
@@ -0,0 +1,57 @@
+package random
+
+import (
+	"testing"
+
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/hashicorp/go-uuid"
+	"github.com/stretchr/testify/require"
+)
+
+func Uuid(t *testing.T) string {
```

Comment:
my friend, this should be UUID

Reply:
I think we have another discussion coming. :D I strongly prefer having abbreviations written in only the first letter capitalized, but you are right; for now, our codebase has them capitalized, so I will rename them.

Reply:
Done.